src/overlays/actors/ovl_Boss_Ganon/z_boss_ganon.h

```diff
@@ -8,13 +8,141 @@ struct BossGanon;
 
 typedef void (*BossGanonActionFunc)(struct BossGanon*, GlobalContext*);
 
+typedef enum {
+    /* 0 */ FWORK_0,
+    /* 1 */ FWORK_1,
+    /* 2 */ DORF_CENTER_POS,
+    /* 3 */ DORF_TRIFORCE_PRIM_B,
+    /* 4 */ DORF_TRIFORCE_PRIM_A,
+    /* 5 */ DORF_TRIFORCE_ENV_G,
+    /* 6 */ DORF_TRIFORCE_SCALE,
+    /* 7 */ DORF_VORTEX_ALPHA,
+    /* 8 */ DORF_VORTEX_SCALE,
+    /* 9 */ FWORK_UNUSED_9,
+    /* 10 */ FWORK_MAX
```

Comment:
(one more reason to use hex if we're going to have comments)

Reply:
lots of enum values have no meaning in hex though
I see the enum comments as a short term solution to help with documentation efforts, and a later point can be removed when we wont be as concerned about their actual value in the list

Reply:
That's fair for specific enums like these which aren't used as actual values, but my main issue is almost all enums are better as hex since that's what we see in hex or asm diffs for instance (which is very relevant when working on other versions among other things), and we have started using decimal more and more for these which is unfortunate imo

Reply:
yea, I realized that after posting my reply. The point of the comments is for how you would see them in a diff, or mips2c, or even in ram in pj64. So thats a fair point.
I can change these, but as you mention we already have a bunch in decimal in the codebase already

Reply:
At this point it would probably not be worth changing them to hex yeah, we can always do manual conversions for the few cases when we need to and that's most likely going to be less time/effort overall